docs/sources/dashboards/variables/variable-syntax/index.md

```diff
@@ -176,14 +176,24 @@ Interpolation result: "'test''1','test2'"
 
 ### Text
 
-Formats single- and multi-valued variables into their text representation. For a single variable it will just return the text representation. For multi-valued variables it will return the text representation combined with `+`.
+Formats single- and multi-valued variables into their text representation. For a single variable it will just return the text representation. For multi-valued variables it will return the text representation combined with `+`. For key-value variables (i.e. 'test1 : 3') it will return the key.
 
 ```bash
 servers = ["test1", "test2"]
 String to interpolate: '${servers:text}'
 Interpolation result: "test1 + test2"
 ```
 
+### Value (Key-value variables)
+
+Returns the value of a key-value variable.
+
+```bash
+servers = ["test1 : 3"]
+String to interpolate: '${servers:value}'
```

Comment:
🤔 we don't support this `value` format, are you using a fork of Grafana with this implementation? 

In the screen recording, you will see the console prints an error and then it fallbacks to the global format, which in the case of `key-value` variables returns the value. 

https://github.com/grafana/grafana/assets/239999/8e2c0b75-7034-49aa-a9f5-79a8026cbbd8